shared/app/globals.native.tsx

```diff
@@ -5,7 +5,7 @@
 //
 // Needed for purepack
 // @ts-ignore
-window.Buffer = require('buffer').Buffer
+globalThis.buffer = global.Buffer = window.Buffer = require('buffer/').Buffer
```

Comment:
Is the trailing slash needed? If so why?

Reply:
yes https://github.com/feross/buffer#usage